synapsis/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Not sure what this is, is it necessary or a mistake?

Reply:
This is a configuration to Mockito to allow us to mock classes that have been marked as final by any framework we use.